### PR TITLE
Instructor: view results: add missing space and icon for 'Edit' link #8921 

### LINKED ISSUE #8921: Instructor: view results: add missing space and icon for 'Edit' link
Context: an instructor viewing results of a session

Current:
image

Suggestion:
Add a space before the `[Edit]` link. We can also add an icon before the word `Edit` too.

### PR DESCRIPTION
**Pull request by ah91086**
Fixes #8921 s.ToReview Changes made again

**PR Checklist**

Ensure that you have:
- [x] Read and understood our PR guideline: https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr
  - [x] Added the issue number to the "Fixes" keyword above
  - [x] Titled the PR as specified in the abovementioned document
- [x] Made your changes on a branch other than `master` and `release`
- [x] Gone through all the changes in this PR and ensured that:
  - [x] They addressed one (and only one) issue
  - [x] No unintended changes were made
- [x] Run and passed static analysis: `./gradlew lint` and `npm run lint`
- [x] Added/updated tests, if changes in functionality were involved
- [x] Added/updated documentation to public APIs (classes, methods, variables), if applicable

**Outline of Solution**

Appended a space character to the end of the 'feedbackSessionName' variable in the InstructorFeedbackResultsSessionPanel class.

### CODE REVIEW

**Comment by ah91086** [comment]
I really appreciate the guidance Shradheya!

**Comment by damithc** [comment]
> This doesn't look good actually. I think we should skip the icon and just add the space only. Having the icon and text on different lines don't look good.

Yes, if at all, it should be in the same line and there should be a space between the session name and the icon. May be show just the icon and show text as a tool tip?

**Comment by ah91086** [comment]
I set the tool tip to be "Edit", but i just found const.FEEDBACK_SESSION_EDIT. Should I change it to that?

File: src/main/webapp/WEB-INF/tags/instructor/results/sessionPanel.tag [hunk 1]
```diff
@@ -21,8 +21,10 @@
             <label class="col-lg-4 control-label">Session:</label>
             <div class="col-lg-8">
               <p class="form-control-static">${sessionPanel.feedbackSessionName}
-                <c:if test="${not empty sessionPanel.editLink}">
-                  <a href="${sessionPanel.editLink}">[Edit]</a>
+                <c:if test="${not empty sessionPanel.editLink}">&nbsp
+                  <a href="${sessionPanel.editLink}">
+                    <span class="glyphicon glyphicon-edit" data-original-title="Edit feedback session details" data-placement="top" data-toggle="tooltip" id="edit_icon_button"></span>
```
Comment (tshradheya) [inline src/main/webapp/WEB-INF/tags/instructor/results/sessionPanel.tag]: I see that you changed it to the content of `Const.Tooltips.FEEDBACK_SESSION_EDIT`. So why not reference the constant here directly

```jsp
data-original-title="<%= Const.Tooltips.FEEDBACK_SESSION_EDIT %>"
```

Comment (tshradheya) [reply src/main/webapp/WEB-INF/tags/instructor/results/sessionPanel.tag]: If you do like this with same indentation, you will follow the correct indentation style and also will not have to add an explicit space

screen shot 2018-07-02 at 4 28 27 pm

Comment (ah91086) [reply src/main/webapp/WEB-INF/tags/instructor/results/sessionPanel.tag]: I attempted to do something in this nature, but I didn't know how.

**Comment by ah91086** [comment]
ready for review again

File: src/main/webapp/WEB-INF/tags/instructor/results/sessionPanel.tag [hunk 2]
```diff
@@ -21,8 +21,14 @@
             <label class="col-lg-4 control-label">Session:</label>
             <div class="col-lg-8">
               <p class="form-control-static">${sessionPanel.feedbackSessionName}
-                <c:if test="${not empty sessionPanel.editLink}">
-                  <a href="${sessionPanel.editLink}">[Edit]</a>
+                <c:if test="${not empty sessionPanel.editLink}">&nbsp
+                  <a href="${sessionPanel.editLink}"
+                     id="edit_icon_button"
```
Comment (tshradheya) [inline src/main/webapp/WEB-INF/tags/instructor/results/sessionPanel.tag]: For line 26-29 follow 4 space indentation as shown here in the image

https://github.com/TEAMMATES/teammates/pull/8923#discussion_r199420404

**Comment by ah91086** [comment]
okay i think this time it is finally free of errors. ready for review.

**Comment by damithc** [comment]
@ah91086 can post a screenshot of the latest UI?

**Pull request merged**